torch/csrc/jit/symbolic_script.cpp

```diff
@@ -154,6 +154,14 @@ const std::vector<std::string> functions = {
 
             return torch.var(self, dim, unbiased, keepdim), backward
 
+        def tanh(self):
+            output = torch.tanh(self)
+            def backward(grad_output):
+                grad_self = torch.tanh_backward(grad_output, output)
+                return grad_self
+
+            return output, backward
```

Comment:
nit: no need to assign `output` and `grad_self`. You should just be able to put the expressions in the `return` lines